package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -201,6 +203,7 @@ _proto_mbim_setup() {
 			json_close_array
 			proto_add_dynamic_defaults
 			[ -n "$zone" ] && json_add_string zone "$zone"
+			[ -n "$ip4table" ] && json_add_string ip4table "$ip4table"
```

Comment:
Does this still work with nftables?

Reply:
This handles routing rules, and goes through netifd's abstraction, so I see no reason not to if it works for uqmi.